command/claw.py

```diff
@@ -0,0 +1,75 @@
+import math
+import config
+from commands2 import SequentialCommandGroup
+from robotpy_toolkit_7407.command import SubsystemCommand
+
+from robot_systems import Robot
+from subsystem import Claw
+
+class ClawInit(SubsystemCommand[Claw]):
+
+    def __init__(self, subsystem: Claw):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+
+    def initialize(self) -> None:   
+        self.subsystem.zero()
+
+    def execute(self) -> None:
+        pass
+
+    def isFinished(self) -> bool:
+        return round(math.degrees(self.subsystem.get_angle())) == 0
+    
+    def end(self, interrupted: bool) -> None:
+        pass
+
+class ClawWrist(SubsystemCommand[Claw]):
+
+    def __init__(self, subsystem: Claw, up: bool):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+        self.up = up
+
+    def initialize(self) -> None:
+        self.subsystem.set_claw_ouput(config.claw_motor_speed)
+        if self.up:
+            self.subsystem.set_angle(math.pi / 2)
+        else:
+            self.subsystem.set_angle(0)
+
+    def execute(self) -> None:
+        pass
+
+    def isFinished(self) -> bool:
+        if self.up:
+            return round(math.degrees(self.subsystem.get_angle())) == 90
```

Comment:
This doesn't need any changes now, but something that you might encounter while tuning automodes is that this command takes a while to finish because 1 degree is a tight tolerance. If you encounter that you can change to `return abs(value-target) < tolerance`.